### PR TITLE
Add service id option to Grid CLI

### PR DESCRIPTION
**Pull request by eloaverona**
Adds service id option for all agents, organization, schema and products CLI commands.

### CODE REVIEW

File: cli/src/main.rs [hunk 1]
```diff
@@ -62,6 +62,8 @@ fn run() -> Result<(), CliError> {
         (@arg wait: --wait +takes_value "How long to wait for transaction to be committed")
         (@arg key: -k +takes_value "base name for private key file")
         (@arg verbose: -v +multiple "Log verbosely")
+        (@arg service_id: --service_id +takes_value "If running on splinter, provide the ID for \
+            the service the payload should be sent to")
```
Comment (ltseeley) [inline cli/src/main.rs]: The help message should probably be formatted like: "The ID of the service the payload should be sent to; required if running on splinter."

File: cli/src/main.rs [hunk 2]
```diff
@@ -62,6 +62,8 @@ fn run() -> Result<(), CliError> {
         (@arg wait: --wait +takes_value "How long to wait for transaction to be committed")
         (@arg key: -k +takes_value "base name for private key file")
         (@arg verbose: -v +multiple "Log verbosely")
+        (@arg service_id: --service_id +takes_value "If running on splinter, provide the ID for \
```
Comment (chenette) [inline cli/src/main.rs]: Splinter (per capitalization guidelines).

Comment (eloaverona) [reply cli/src/main.rs]: updated

File: cli/src/main.rs (hunk 1 above)
Comment (eloaverona) [reply cli/src/main.rs]: updated

[the remark this answers is not in the document]
Comment (eloaverona) [reply cli/src/main.rs]: Would this be confusing? I thought circuit-id is part of the service-id. `<circuit-id>::<service-id>` seems to imply they are separate things the user has to put together.

[the remark this answers is not in the document]
Comment (eloaverona) [reply cli/src/main.rs]: Ok, looking at Andi's example at https://github.com/Cargill/splinter/pull/414 it seems they are separate things the user has to put together. Given that, to call this argument `service-id` might be confusing?

[the remark this answers is not in the document]
Comment (eloaverona) [reply cli/src/main.rs]: I added the "<circuit-id>::<service-id>" format example for now

**Pull request merged**